api4/post.go

```diff
@@ -12,6 +12,11 @@ import (
 	"github.com/mattermost/mattermost-server/model"
 )
 
+const (
+	PAGE_DEFAULT     = 0
+	PER_PAGE_DEFAULT = 60
```

Comment:
Added this constants instead of using those available at https://github.com/mattermost/mattermost-server/blob/scrolling-overhaul/web/params.go#L15-L16